### DISCUSSION TITLE
Get current package.json version

### DISCUSSION THREAD

**Discussion by nephix** [facebook/docusaurus#5198] [Q&A] [answered]
I would like to access the current `version` prop of the `package.json`. Something like

```
---
sidebar_position: 0
---

import packageJson from "../package.json";

const Version = () => {
	return (<span>packageJson.version</span>);
}
```
in an `mdx` file and then use `<Version />` within the markdown. I probably made a mistake or it's not possible. Anybody have an idea?

**Comment by slorber** [comment, marked as answer]
You should be able to do that, considering the import path is correct.
If this does not work you probably have an error message to share.

```mdx
---
sidebar_position: 0
---

import packageJson from "../package.json";

<span>{packageJson.version}</span>
```

**Reply by nephix** [reply]
Ok you are right it works, my mistake was that I did

```
<span>{packageJson.version}</span>
```

within a code (```) block. Is there a way to do it within a code block?

**Reply by slorber** [reply]
you can use `<code>` for inline code blocks.

not sure it's related but you can also display the content of that json file as a json code block: https://docusaurus.io/examples/markdownPageExample#import-code-block-from-source-code-file
Docusaurus don't provide (yet) first-class support for it, so it's for now a feature of Webpack on which we won't provide support, but it remains useful to know.

**Reply by nephix** [reply]
Thanks so much 🙏

**Reply by nephix** [reply]
I'm new to discussions, is there a way to mark it as resolved?

**Reply by slorber** [reply]
![image](https://user-images.githubusercontent.com/749374/126464906-9f3089bd-ddcb-4d1e-9eb0-b2c0dde85089.png)
